nemo_skills/pipeline/convert.py

```diff
@@ -82,17 +82,24 @@ def get_hf_to_trtllm_cmd(
 def get_hf_to_nemo_cmd(
     input_model, output_model, model_type, hf_model_name, dtype, num_gpus, num_nodes, extra_arguments
 ):
+    # Check if the model_type is "nemo"
+    if model_type == "qwen":
+        conversion_command = "python -m nemo_skills.conversion.hf_to_nemo_qwen"
```

Comment:
Let's either change this to be with just `script = ht_to_nemo_qwen` same as for trtllm or change the other one to have this conversion_command style. Either way works, but it's best to keep things consistent